### PR TITLE
Add initial variables for red and blue colors.

### PR DESCRIPTION
**Pull request by lynnerang**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

`This is a small PR to add initial variables for blue and red in the color scheme in case others want to use them.`

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:
`Your message here`

## Has This Been Tested?

- [x] No
- [ ] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
`Your message here`
- [ ] Have any prior tests been changed?
If so, please explain:
`Your message here`  

# Additional notes:
`Your message here`

# Checklist:

- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas

Co-Authored-By: Justin Pyktel <justinpyktel@gmail.com>
Co-Authored-By: Lynne Rang <lynne.rang@gmail.com>

### CODE REVIEW

File: src/components/App/_App.scss [hunk 1]
```diff
@@ -1,6 +1,4 @@
-// .App {
-
-// }
+@import '../../variables.scss';
```
Comment (SiimonStark) [inline src/components/App/_App.scss:1]: Do we need this here?? I have a variable file in my PR
I already grabbed many of the colors from the Figma Mock up

File: src/components/Score/_Score.scss [hunk 2]
```diff
@@ -24,13 +24,13 @@
   }
 
   &.blue-score {
-    background: #5861B4;
-    border: 4px solid rgb(36, 36, 108);
+    background: $blue;
+    border: 4px solid $dark-blue;
```
Comment (SiimonStark) [inline src/components/Score/_Score.scss:28]: we'll need to check the variables I setup, I gave them pretty descriptive variable names

File: src/variables.scss [hunk 3]
```diff
@@ -0,0 +1,4 @@
+$blue: #5861B4;
```
Comment (SiimonStark) [inline src/variables.scss:1]: Again, I already have a variable file. And it's linked to the `index.scss` file

**Review by SiimonStark — CHANGES_REQUESTED** [review]
I like the thought behind it, but I should already have those setup

File: src/components/App/_App.scss (hunk 1 above)
Comment (lynnerang) [reply src/components/App/_App.scss:1]: Well it wasn't merged yet, so this is just a conflict of our two separate branches we'll need to address.

**Pull request closed without merging**